### DISCUSSION TITLE
I have everything set up and working except for the "Delivery Message Sensor". I am confused on it's purpose.

### DISCUSSION THREAD

**Discussion by liquidbear99** [moralmunky/Home-Assistant-Mail-And-Packages#579] [Q&A]
What is the template sensor for the "delivery message sensor" for?

**Comment by firstof9** [comment]
It's not required, someone requested an easy way to parse the information into a sensor to display on their frontend.

The same template can be used to generate TTS information for announcements and/or notification messages.

**Comment by Thecodeman35** [comment]
Would it be possible to explain how to add this feature? I like the additional text but I don't see this as an option in the sensor list.

**Reply by moralmunky** [reply]
See the WIKI section [Mail Summary Message](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/Mail-Summary-Message)
